### ISSUE TITLE
How can I choose the optimal checkpoint in the pre-training model

### ISSUE THREAD

**Issue by liutongyang** [salesforce/BLIP#49]
In the pre-training mode, each epoch is saved. 

It seems that the code does not filter the optimal model by the accuracy or loss value. 

What is your strategy?

**Comment by liutongyang** [comment]
Thanks for your answer.

**Comment by liutongyang** [comment]
By the way, 

is finetune's checkpoint based on the 20th epoch of pre-training reported in the paper?

**Comment by LiJunnan1992** [comment]
Yes

**Comment by liutongyang** [comment]
thanks a lot.
